tests/unittest/_torch/speculative/hw_agnostic/test_dflash_accept_stats.py

```
@@ -0,0 +1,271 @@
+hosts without a GPU stack; the fallback loader below keeps them runnable
+even where the tensorrt_llm package itself cannot be imported.
+"""
+
+import json
+import os
+
+import pytest
+
+try:
+    from tensorrt_llm._torch.speculative import accept_stats
+except Exception:  # pragma: no cover - torch-less host fallback
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)